.gitignore

```diff
@@ -11,3 +11,5 @@ vendor
 default*
 *.bak
 .vscode/
+/.dashboard_asset_cache
```

Comment:
need delete？

Reply:
If I use the master check to the 3.0 branch, is's will be new files.